webpack/helpers.js

```diff
@@ -4,6 +4,31 @@ function root(path) {
   return resolve(__dirname, '..', path);
 }
 
+function checkNewVersion(options, appVersion){
+  let http = require('http');
+  let semver = require('semver');
+  let notifier = require('node-notifier');
+
+  http.get(options, function(resp){
+    let body = '';
+    resp.on('data', function(chunk){
+      body += chunk;
+    });
+    resp.on('end', function(){
+      let parsed = JSON.parse(body);
+      if (semver.gt(parsed.message, appVersion)){
+        notifier.notify({
+          'title': `New version is available!`,
+          'message': `bterm version ${parsed.message} is available.`
```

Comment:
Would be better if there would be an attribute `version` with value `parsed.message`. Then you decide on the front end what kind of message you display.